app/controllers/api_controller.rb

```diff
@@ -262,6 +262,10 @@ def included_relationships
     @included_relationships ||= include_params_handler.included_relationships
   end
 
+  def meta_fields
+    @meta_fields ||= include_params_handler.meta_fields
```

Comment:
It might be slightly nicer to return an empty array here rather than `nil` if no params were passed, that then makes using this easier without a nil guard in `index_and_render`. Or if that doesn't appeal then perhaps add another method e.g.

```ruby
def meta_include?(name)
  meta_fields&.include?(name.to_s)
end
```